serialization/api.go

```diff
@@ -82,334 +82,334 @@ type Serializer interface {
 	// Id returns id of serializer.
 	Id() int32
 
-	// Read reads object from ObjectDataInput.
+	// Read reads an object from ObjectDataInput.
 	Read(input DataInput) (interface{}, error)
 
-	// Write writes object to ObjectDataOutput.
+	// Write writes an object to ObjectDataOutput.
 	Write(output DataOutput, object interface{}) error
 }
 
 // IData is the basic unit of serialization. It stores binary form of an object serialized
 // by SerializationService's ToData() method.
 type IData interface {
-	// Returns byte array representation of internal binary format.
+	// Buffer returns byte array representation of internal binary format.
 	Buffer() []byte
 
-	// Returns serialization type of binary form.
+	// GetType returns serialization type of binary form.
 	GetType() int32
 
-	// Returns the total size of Data in bytes.
+	// TotalSize returns the total size of Data in bytes.
 	TotalSize() int
 
-	// Returns size of internal binary data in bytes.
+	// DataSize returns size of internal binary data in bytes.
 	DataSize() int
 
-	// Returns partition hash calculated for serialized object.
+	// GetPartitionHash returns partition hash calculated for serialized object.
 	GetPartitionHash() int32
 }
 
 // DataOutput provides serialization methods.
 type DataOutput interface {
-	// Returns the head position in the byte array.
+	// Position returns the head position in the byte array.
 	Position() int32
 
-	// Sets the head position in the byte array.
+	// SetPosition sets the head position in the byte array.
```

Comment:
This is a built-in Go method and its GoDoc.
```
// Return the list of export search paths for this GccgoInstallation.
func (inst *GccgoInstallation) SearchPaths() (paths []string) {
```

Reply:
The convention is to start with the method name